pyproject.toml

```diff
@@ -17,9 +17,10 @@ include = [
 ]
 exclude = [
     "build/**",
+    "docs/source/conf.py",
+    "docs/source/examples/text/enwiki_tok/**",
```

Comment:
There is no such path `docs/source/examples/text/enwiki_tok/**`.

Reply:
Ah, `docs/source/examples` is a softlink to `examples/`.

When I remove that line and run `pre-commit run --all-files`, I get a ton of spew that we want to not mess with originating in `docs/source/examples/text/enwiki_tok/`.